### PR TITLE
cmdline refactor

### PR DESCRIPTION
**Pull request by danilexn**
There are some arguments that are inconsistent between commands, namely:

- `-c [--container]`
- `-rf [--renderingFactor]`

In some commands, these were the old `-i` and `-sf`. I propose to harmonize for user-friendliness

### CODE REVIEW

**Comment by danilexn** [comment]
I am working on the wiki with the latest API, including these changes and the new section for `st-align-interactive`.

File: src/main/java/cmd/InteractiveAlignment.java [hunk 1]
```diff
@@ -60,7 +60,7 @@
 //-i /Users/preibischs/Documents/BIMSB/Publications/imglib2-st/slide-seq/raw/slide-seq.n5 -d1 Puck_180602_20.n5 -d2 Puck_180602_18.n5 -n 8 -sk 2 --ffSingleSpot 1.5
 public class InteractiveAlignment implements Callable<Void> {
 
-	@Option(names = {"-i", "--input"}, required = true, description = "input N5 container path, e.g. -i /home/ssq.n5.")
+	@Option(names = {"-c", "--container"}, required = true, description = "input N5 container path, e.g. -i /home/ssq.n5.")
```
Comment (minnerbe) [inline src/main/java/cmd/InteractiveAlignment.java:63]: Can you please also change the parameter in the description, i.e., `-c /home/ssq.n5`?

File: src/main/java/cmd/PairwiseSectionAligner.java [hunk 2]
```diff
@@ -35,7 +35,7 @@
 @Command(name = "st-align-pairs", mixinStandardHelpOptions = true, version = "0.3.0", description = "Spatial Transcriptomics as IMages project - align pairs of slices")
 public class PairwiseSectionAligner implements Callable<Void> {
 
-	@Option(names = {"-i", "--input"}, required = true, description = "input N5 container path, e.g. -i /home/ssq.n5.")
+	@Option(names = {"-c", "--container"}, required = true, description = "input N5 container path, e.g. -i /home/ssq.n5.")
```
Comment (minnerbe) [inline src/main/java/cmd/PairwiseSectionAligner.java:38]: See above

**Review by minnerbe — CHANGES_REQUESTED** [review]
Having `-i` instead of `-c` was intentional. However, the logic behind that was pretty flawed, so having `-c` is definitely better, thanks! :+1:

**Pull request merged**